Fix internal error if new meeting is created

If a new meeting is created the following Error is thrown: 
```
ERROR Error: Uncaught (in promise): Error: <html>
  <head>
    <title>Internal Server Error</title>
  </head>
  <body>
    <h1><p>Internal Server Error</p></h1>
    
  </body>
</html>

Error: <html>
  <head>
    <title>Internal Server Error</title>
  </head>
  <body>
    <h1><p>Internal Server Error</p></h1>
    
  </body>
</html>
```

**Reproduction:**
1. Create a new committee
2. start creating a new meeting 
3. press save
4. error is thrown

```
DEBUG:datastore:HTTP error 400: {'type': 3, 'fqid': 'meeting/4', 'type_verbose': 'MODEL_DOES_NOT_EXIST'}
backend_1           | DEBUG:openslides_backend.services.datastore.adapter: Model 'meeting/4' does not exist.
backend_1           | 2024-02-01T17:31:54.724: [/usr/local/lib/python3.10/site-packages/datastore/shared/flask_frontend/errors.py] [DEBUG] HTTP error 400: {'type': 3, 'fqid': 'meeting/4', 'type_verbose': 'MODEL_DOES_NOT_EXIST'}
backend_1           | [2024-02-01 17:31:54 +0000] [59] [ERROR] Error handling request /system/action/handle_request
backend_1           | Traceback (most recent call last):
backend_1           |   File "/usr/local/lib/python3.10/site-packages/gunicorn/workers/gthread.py", line 282, in handle
backend_1           |     keepalive = self.handle_request(req, conn)
backend_1           |   File "/usr/local/lib/python3.10/site-packages/gunicorn/workers/gthread.py", line 334, in handle_request
backend_1           |     respiter = self.wsgi(environ, resp.start_response)
backend_1           |   File "/app/openslides_backend/http/application.py", line 103, in __call__
backend_1           |     return self.wsgi_application(environ, start_response)
backend_1           |   File "/app/openslides_backend/http/application.py", line 93, in wsgi_application
backend_1           |     response = self.dispatch_request(request)
backend_1           |   File "/app/openslides_backend/http/application.py", line 44, in dispatch_request
backend_1           |     response_body, access_token = view_instance.dispatch(request)
backend_1           |   File "/app/openslides_backend/http/views/base_view.py", line 119, in dispatch
backend_1           |     return func(request)
backend_1           |   File "/app/openslides_backend/http/views/action_view.py", line 47, in action_route
backend_1           |     response = handle_action_in_worker_thread(
backend_1           |   File "/app/openslides_backend/action/action_worker.py", line 69, in handle_action_in_worker_thread
backend_1           |     raise action_worker_thread.exception
backend_1           |   File "/app/openslides_backend/action/action_worker.py", line 242, in run
backend_1           |     self.response = self.handler.handle_request(
backend_1           |   File "/app/openslides_backend/action/action_handler.py", line 115, in handle_request
backend_1           |     results = self.execute_write_requests(self.parse_actions, payload)
backend_1           |   File "/app/openslides_backend/action/action_handler.py", line 156, in execute_write_requests
backend_1           |     write_requests, data = get_write_requests(*args)
backend_1           |   File "/app/openslides_backend/action/action_handler.py", line 191, in parse_actions
backend_1           |     write_request, results = self.perform_action(
backend_1           |   File "/app/openslides_backend/action/action_handler.py", line 240, in perform_action
backend_1           |     write_request, results = action.perform(
backend_1           |   File "/app/openslides_backend/action/action.py", line 182, in perform
backend_1           |     instance = self.base_update_instance(instance)
backend_1           |   File "/app/openslides_backend/action/actions/meeting/create.py", line 72, in base_update_instance
backend_1           |     return super().base_update_instance(instance)
backend_1           |   File "/app/openslides_backend/action/mixins/create_action_with_dependencies.py", line 18, in base_update_instance
backend_1           |     instance = super().base_update_instance(instance)
backend_1           |   File "/app/openslides_backend/action/generics/create.py", line 29, in base_update_instance
backend_1           |     instance = self.update_instance(instance)
backend_1           |   File "/app/openslides_backend/action/actions/meeting/create.py", line 90, in update_instance
backend_1           |     self.check_start_and_end_time(instance)
backend_1           |   File "/app/openslides_backend/action/actions/meeting/mixins.py", line 62, in check_start_and_end_time
backend_1           |     if start_time > end_time:
backend_1           | TypeError: '>' not supported between instances of 'NoneType' and 'NoneType'
auth_1              | 2024-02-01T17:32:04.368Z: Metric: {"sessions":2,"users":2}

```

--- a/openslides_backend/action/actions/meeting/mixins.py
+++ b/openslides_backend/action/actions/meeting/mixins.py
@@ -47,19 +47,11 @@ class MeetingCheckTimesMixin(Action):
                 ["start_time", "end_time"],
                 raise_exception=False,
             )
-        start_time = (
-            instance["start_time"]
-            if "start_time" in instance
-            else db_instance.get("start_time")
-        )
-        end_time = (
-            instance["end_time"]
-            if "end_time" in instance
-            else db_instance.get("end_time")
-        )
+        start_time = instance.get("start_time", db_instance.get("start_time"))
+        end_time = instance.get("end_time", db_instance.get("end_time"))
         if start_time and not end_time or not start_time and end_time:
             raise ActionException("Only one of start_time and end_time is not allowed.")
-        if start_time > end_time:
+        if start_time and end_time and start_time > end_time:
             raise ActionException("start_time must be before end_time.")
 
 

--- a/tests/system/action/meeting/test_create.py
+++ b/tests/system/action/meeting/test_create.py
@@ -305,6 +305,9 @@ class MeetingCreateActionTest(BaseActionTestCase):
             set_400_str="Only one of start_time and end_time is not allowed.",
         )
 
+    def test_create_empty_times(self) -> None:
+        self.basic_test({"start_time": None, "end_time": None})
+
     def test_create_name_too_long(self) -> None:
         self.basic_test(
             {"name": "A" * 101},

